oms-active-directory-security-audit-solution/azuredeploy.json

```diff
@@ -0,0 +1,743 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Specify the workspace name"
+            }
+        },
+        "workspaceRegionId": {
+            "type": "string",
+            "metadata": {
+                "description": "Specify the workspace region"
+            }
+        },
+        "pricingTier": {
+            "type": "string",
+            "defaultValue": "Standalone",
+            "allowedValues": [
+                "Free",
+                "Standalone",
+                "PerNode"
+            ],
+            "metadata": {
+                "description": "Service Tier: Free, Standalone, or PerNode"
+            }
+        }
+    },
+    "variables": {
+       
+        "LogAnalyticsApiVersion": "2017-03-03-preview",
+        "SolutionVersion": "1.1",
+        "SolutionName": "AD Security Audit Solution",
+        "SolutionPublisher": "Asitha De Silva",
+        "ViewAuthor": "asithahttp@gmail.com",

```

Comment:
looks like many of these are not addressed, go back through the original comments and make sure you have each one covered

Reply:
removed